proto/minder/v1/minder.proto

```diff
@@ -1810,13 +1810,25 @@ message ListEvaluationResultsRequest {
     // an empty label_filter.
     oneof profile_selector {
         // ID can contain either a profile name or an ID
-        string profile = 2;
+        string profile = 2 [
+            (buf.validate.field).string = {
+                pattern: "^([[:alnum:]][-[:word:]]*)?$"
+                max_len: 200,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
         // Filter profiles to only those matching the specified labels.
         // 
         // The default is to return all user-created profiles; the string "*" can
         // be used to select all profiles, including system profiles.  This syntax
         // may be expanded in the future.
-        string label_filter = 3;
+        string label_filter = 3 [
+            (buf.validate.field).string = {
+                pattern: "^(\\*|[a-zA-Z][a-zA-Z0-9_]*)$",
```

Comment:
This doesn't seem to be used at all on the server side, so I'm not sure what we intended here.  I'm assuming this is a single word-type label, which is what you've got.

Reply:
It was that and `*` too. But yeah, there were a few places where the fields were there without being used.